lib/internal/modules/cjs/loader.js

```diff
@@ -983,8 +982,17 @@ Module.prototype.load = function(filename) {
 
   const extension = findLongestRegisteredExtension(filename);
   // allow .mjs to be overridden
-  if (StringPrototypeEndsWith(filename, '.mjs') && !Module._extensions['.mjs'])
-    throw new ERR_REQUIRE_ESM(filename);
+  if (StringPrototypeEndsWith(filename, '.mjs') &&
+    !Module._extensions['.mjs']) {
+    process.emitWarning('Attempting to require and ESModule.' +
```

Comment:
```suggestion
    process.emitWarning('Attempting to require and ESModule. ' +
```
[Tests](https://github.com/nodejs/node/runs/2851669236?check_suite_focus=true)